### PR TITLE
[Do Not Merge] Org id keep denormalize merge

### PR DESCRIPTION
**Pull request by Colin-Lane**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/model/Keep.scala [hunk 1]
```diff
@@ -85,45 +86,69 @@ object Keep {
     }
   }
 
+  def applyFromDbRowTuples(firstArguments: KeepFirstArguments, restArguments: KeepRestArguments): Keep = (firstArguments, restArguments) match {
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala:89]: Hah, what a weird world we're in, where this is necessary. I'm sorry.

Comment (Colin-Lane) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala:89]: I checked for alternatives, it was just as hard to write as it is to look at.

File: kifi-backend/modules/common/conf/evolutions/shoebox/342.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Shoebox
+
+# --- !Ups
+
+ALTER TABLE bookmark
+	ADD COLUMN organization_id bigint(20) DEFAULT NULL;
+ALTER TABLE bookmark
+    ADD CONSTRAINT `bookmark_f_organization_id` FOREIGN KEY (`organization_id`) REFERENCES organization(`id`);
+
```
Comment (Colin-Lane) [inline kifi-backend/modules/common/conf/evolutions/shoebox/342.sql:9]: @andrewconner @leogrim Going to apply this evolution tonight some time unless there are objections

Comment (leogrim) [inline kifi-backend/modules/common/conf/evolutions/shoebox/342.sql:9]: No constraint at least for now, we're not honoring it.

**Comment by leogrim** [comment]
There's now a conflict on evolutions, you can get rid of this one, I have added a `343.sql` and applied it on my own. Closing this PR so that you can fix the conflicts.

**Pull request closed without merging**